commands.c

```diff
@@ -63,6 +63,8 @@ bool cg_quit(arg_t _)
 		}
 	}
 	exit(EXIT_SUCCESS);
+	// silence complier warning
+	return 0;
```

Comment:
Which compiler are you using? I don't get any warning on either GCC or Clang.

Reply:
Perhaps he is using tcc or something similar, iirc just like all suckless tools sxiv can be compiled with tcc.

Reply:
Yes I was using tcc. Don't like getting into the habit of ignoring compiler messages so just added the (useless) line. Feel free to remove if it bothers you.